lib/modules/asset/types/AssetGroupContent.ts

```diff
@@ -0,0 +1,9 @@
+import { KDocumentContent } from "kuzzle-sdk";
+
+export interface AssetsGroupsBody {
+  name: string;
+  children: string[];
+  parent: string | null;
```

Comment:
parent?: string;

Reply:
What do you want ?
You want to define `parent` undefinable instead nullable ?